bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -69,12 +70,9 @@ spec:
               ovs-vsctl --no-wait set Open_vSwitch . other_config:n-revalidator-threads=4
               ovs-vsctl --no-wait set Open_vSwitch . other_config:n-handler-threads=10
           fi
-          /usr/share/openvswitch/scripts/ovs-ctl start --no-ovsdb-server --system-id=random
+          /usr/share/openvswitch/scripts/ovs-ctl load-kmod
 
-          tail --follow=name /var/log/openvswitch/ovs-vswitchd.log /var/log/openvswitch/ovsdb-server.log &
-          while true; do
-            sleep 10000
-          done
+          exec ovs-vswitchd unix:/var/run/openvswitch/db.sock -vconsole:info -vsyslog:warn --mlockall --no-chdir --pidfile=/var/run/openvswitch/ovs-vswitchd.pid --syslog-method=unix:/dev/log
```

Comment:
shouldn't we be passing the same value for `-vconsole` and `-vsyslog`?

Reply:
I was looking in to that, and ovs-vswitchd is spammy and it's info messages are basically useless.

Reply:
ok, so shouldn't we do `-vconsole:warn' then?

Reply:
On second thought, we might as well just set them both to info.